ccd-definition/CaseEventToFields/DefendantResponseLRspec.json

```diff
@@ -565,7 +573,6 @@
     "DisplayContext": "COMPLEX",
     "PageID": "SmaillClaimHearing",
     "CaseEventFieldLabel": "Hearing",
-    "PageShowCondition": "responseClaimTrack= \"SMALL_CLAIM\" AND specDefenceFullAdmittedRequired!=\"Yes\"",
```

Comment:
SmallClaimHearing screen should only appear for small claim, doesn't it?

Reply:
You are correct, I change wrong "Hearing" for SmallClaims this should shows up, but it should be invisible for fasttrack, cause there is different "Hearing". But I had to remove that "specDefenceFullAdmittedRequired" part of it otherwise it still show up on fast track.